pkg/olm/handler.go

```diff
@@ -71,6 +72,20 @@ func (o *OLMHandler) OperandsList(user *auth.User, w http.ResponseWriter, r *htt
 		return
 	}
 
+	if operandsList != nil && len(operandsList.Items) > 0 {
```

Comment:
Add head comment to explain how this work would help.

For example:
// Deduplicate operands by UID to prevent duplicate CRs

Reply:
lets pull this section into a separate function for better readability.

```go
func DeduplicateUnstructuredList(list *unstructured.UnstructuredList) *unstructured.UnstructuredList {
    seen := make(map[string]bool)
    ...
    list.Items = uniqueItems
    return list
}
```